CONTRIBUTING.md

```diff
@@ -9,12 +9,12 @@ Everyone is welcome to get involved with the project. There are different ways t
       1. using the GitHub discussion at Kornia repo: [GH Discussions](https://github.com/kornia/kornia/discussions)
       2. our slack workspace to keep in touch with our core contributors and the community:
          [Join Here](https://join.slack.com/t/kornia/shared_invite/zt-csobk21g-2AQRi~X9Uu6PLMuUZdvfjA)
-      3. using the `#kornia` tag in [PyTorch Discuss](https://discuss.pytorch.org>)
+      3. using the `#kornia` tag in [PyTorch Discuss](https://discuss.pytorch.org)
```

Comment:
nice catch

Reply:
@johnnv1  - I couldn't use that link myself. What is now missing for the PR to be merged? Your approval or perhaps one more? Some checks are failing but not due to my changes.